rtloader/include/rtloader.h

```diff
@@ -35,7 +35,7 @@ class RtLoader
     RtLoader(cb_memory_tracker_t memtrack_cb);
 
     //! Destructor.
-    virtual ~RtLoader(){};
+    virtual ~RtLoader() {};
```

Comment:
This breaks the linters I think

Reply:
Ah, this change was meant to fix a [linter error](https://app.circleci.com/pipelines/github/DataDog/datadog-agent/127282/workflows/5c35b4a3-2ab0-4d08-bdae-842a79282d25/jobs/1452610) in circle-ci, I suppose we're having mismatching versions with different rules.
The circleci image is using `Ubuntu clang-format version 19.0.0`
while the debian image which runs most of the gitlab linting is using `clang-format version 8.0.0 (tags/RELEASE_800/final)`

Reply:
Both version definitely disagree with each other:
```
root@8a0ab9905e33:/tmp/datadog-agent# clang-format -n rtloader/include/rtloader.h 
rtloader/include/rtloader.h:38:24: warning: code should be clang-formatted [-Wclang-format-violations]
    virtual ~RtLoader(){};
                       ^
```

```
 ~/dd/datadog-agent   chouquette/bump_cryptography  clang-format-15 -n rtloader/include/rtloader.h
rtloader/include/rtloader.h:38:24: warning: code should be clang-formatted [-Wclang-format-violations]
    virtual ~RtLoader() {};
                       ^
```